src/test/java/Homework16.java

```diff
@@ -0,0 +1,33 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+@Test
+public class Homework16 extends BaseTest {
+    @Test
+    public static void registrationNavigation(){
+
+        WebDriver driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(5
+        ));
+
+        String url= "https://bbb.testpro.io/";
+
+        driver.get(url);
+
+        WebElement registrationLink = driver.findElement(By.cssSelector ("[id='hel']"));
+        registrationLink.click();
+
+        String urlRegistration = "https://bbb.testpro.io/Registration.php";
+        Assert.assertEquals(driver.getCurrentUrl(),urlRegistration);
```

Comment:
Test failed! You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/221401468-d3c35cab-0785-47e3-8af1-332d321879d9.png)